pkg/middleware/auth_proxy/auth_proxy.go

```diff
@@ -298,21 +298,18 @@ func (auth *AuthProxy) GetSignedUser(userID int64) (*models.SignedInUser, *Error
 }
 
 // Remember user in cache
-func (auth *AuthProxy) Remember() *Error {
+func (auth *AuthProxy) Remember(id int64) *Error {
+	key := auth.getKey()
 
-	// Make sure we do not rewrite the expiration time
-	if auth.InCache() {
+	// Check if user already in cache
+	userID, _ := auth.store.Get(key)
```

Comment:
I think we should handle this error. 
if its `err` and not `ErrItemNotFound` then we want to know about it.

Reply:
Good point. Will fix